### ISSUE TITLE
ImportError: No module named six

### ISSUE THREAD

**Issue by PatMyron** [brython-dev/brython#1197]
```
brython.js:8724 GET http://localhost:63342/Github/cfn-lint/six.py?v=1566456201210 404 (Not Found)
$download_module @ brython.js:8724
find_spec @ brython.js:8982
method @ brython.js:5227
find_spec @ brython.js:8939
f @ brython.js:6267
method @ brython.js:6271
import_hooks @ brython.js:13220
$B.$__import__ @ brython.js:9028
$B.$import @ brython.js:9061
(anonymous) @ VM151749:42
(anonymous) @ VM151749:6471
run_py @ brython.js:8807
exec_module @ brython.js:8919
f @ brython.js:6267
method @ brython.js:6271
import_hooks @ brython.js:13251
$B.$__import__ @ brython.js:9028
$B.$import @ brython.js:9061
(anonymous) @ VM151744:13
$B.loop @ brython.js:5108
$B.inImported @ brython.js:5098
$B.loop @ brython.js:5114
$B.run_script @ brython.js:4950
$B.loop @ brython.js:5114
req.onreadystatechange @ brython.js:5088
XMLHttpRequest.send (async)
$B.ajax_load_script @ brython.js:5089
$B.loop @ brython.js:5114
idb_cx.onsuccess @ brython.js:5078
IndexedDB (async)
$B.idb_open @ brython.js:5065
$B.loop @ brython.js:5114
$B.parser._run_scripts @ brython.js:5009
$B.parser.brython @ brython.js:4932
onload @ index.html
brython.js:8729 Error 404 means that Python module six was not found at url http://localhost:63342/Github/cfn-lint/six.py
brython.js:8724 GET http://localhost:63342/Github/cfn-lint/six/__init__.py?v=1566456201497 404 (Not Found)
$download_module @ brython.js:8724
find_spec @ brython.js:8982
method @ brython.js:5227
find_spec @ brython.js:8939
f @ brython.js:6267
method @ brython.js:6271
import_hooks @ brython.js:13220
$B.$__import__ @ brython.js:9028
$B.$import @ brython.js:9061
(anonymous) @ VM151749:42
(anonymous) @ VM151749:6471
run_py @ brython.js:8807
exec_module @ brython.js:8919
f @ brython.js:6267
method @ brython.js:6271
import_hooks @ brython.js:13251
$B.$__import__ @ brython.js:9028
$B.$import @ brython.js:9061
(anonymous) @ VM151744:13
$B.loop @ brython.js:5108
$B.inImported @ brython.js:5098
$B.loop @ brython.js:5114
$B.run_script @ brython.js:4950
$B.loop @ brython.js:5114
req.onreadystatechange @ brython.js:5088
XMLHttpRequest.send (async)
$B.ajax_load_script @ brython.js:5089
$B.loop @ brython.js:5114
idb_cx.onsuccess @ brython.js:5078
IndexedDB (async)
$B.idb_open @ brython.js:5065
$B.loop @ brython.js:5114
$B.parser._run_scripts @ brython.js:5009
$B.parser.brython @ brython.js:4932
onload @ index.html
brython.js:8729 Error 404 means that Python module six was not found at url http://localhost:63342/Github/cfn-lint/six/__init__.py
brython.js:8724 GET http://localhost:63342/Github/cfn-lint/Lib/site-packages/six.py?v=1566456201504 404 (Not Found)
$download_module @ brython.js:8724
find_spec @ brython.js:8982
method @ brython.js:5227
find_spec @ brython.js:8939
f @ brython.js:6267
method @ brython.js:6271
import_hooks @ brython.js:13220
$B.$__import__ @ brython.js:9028
$B.$import @ brython.js:9061
(anonymous) @ VM151749:42
(anonymous) @ VM151749:6471
run_py @ brython.js:8807
exec_module @ brython.js:8919
f @ brython.js:6267
method @ brython.js:6271
import_hooks @ brython.js:13251
$B.$__import__ @ brython.js:9028
$B.$import @ brython.js:9061
(anonymous) @ VM151744:13
$B.loop @ brython.js:5108
$B.inImported @ brython.js:5098
$B.loop @ brython.js:5114
$B.run_script @ brython.js:4950
$B.loop @ brython.js:5114
req.onreadystatechange @ brython.js:5088
XMLHttpRequest.send (async)
$B.ajax_load_script @ brython.js:5089
$B.loop @ brython.js:5114
idb_cx.onsuccess @ brython.js:5078
IndexedDB (async)
$B.idb_open @ brython.js:5065
$B.loop @ brython.js:5114
$B.parser._run_scripts @ brython.js:5009
$B.parser.brython @ brython.js:4932
onload @ index.html
brython.js:8729 Error 404 means that Python module six was not found at url http://localhost:63342/Github/cfn-lint/Lib/site-packages/six.py
brython.js:8724 GET http://localhost:63342/Github/cfn-lint/Lib/site-packages/six/__init__.py?v=1566456201511 404 (Not Found)
$download_module @ brython.js:8724
find_spec @ brython.js:8982
method @ brython.js:5227
find_spec @ brython.js:8939
f @ brython.js:6267
method @ brython.js:6271
import_hooks @ brython.js:13220
$B.$__import__ @ brython.js:9028
$B.$import @ brython.js:9061
(anonymous) @ VM151749:42
(anonymous) @ VM151749:6471
run_py @ brython.js:8807
exec_module @ brython.js:8919
f @ brython.js:6267
method @ brython.js:6271
import_hooks @ brython.js:13251
$B.$__import__ @ brython.js:9028
$B.$import @ brython.js:9061
(anonymous) @ VM151744:13
$B.loop @ brython.js:5108
$B.inImported @ brython.js:5098
$B.loop @ brython.js:5114
$B.run_script @ brython.js:4950
$B.loop @ brython.js:5114
req.onreadystatechange @ brython.js:5088
XMLHttpRequest.send (async)
$B.ajax_load_script @ brython.js:5089
$B.loop @ brython.js:5114
idb_cx.onsuccess @ brython.js:5078
IndexedDB (async)
$B.idb_open @ brython.js:5065
$B.loop @ brython.js:5114
$B.parser._run_scripts @ brython.js:5009
$B.parser.brython @ brython.js:4932
onload @ index.html
brython.js:8729 Error 404 means that Python module six was not found at url http://localhost:63342/Github/cfn-lint/Lib/site-packages/six/__init__.py
brython.js:8807 six for module cfnlint
brython.js:8808 module {$class: {…}, __name__: "cfnlint", __doc__: {…}, __package__: "cfnlint", __loader__: {…}, …}
brython.js:8809 $B.parser.$Node {type: "module", children: Array(2), yield_atoms: Array(0), add: ƒ, insert: ƒ, …}
brython.js:8810 six
    at Object._b_.ImportError.$factory (eval at $make_exc (http://localhost:63342/Github/cfn-lint/brython.js:7501:5), <anonymous>:71:321)
    at Object.import_hooks (http://localhost:63342/Github/cfn-lint/brython.js:13227:25)
    at $B.$__import__ (http://localhost:63342/Github/cfn-lint/brython.js:9028:60)
    at Object.$B.$import (http://localhost:63342/Github/cfn-lint/brython.js:9061:43)
    at eval (eval at run_py (http://localhost:63342/Github/cfn-lint/brython.js:8807:14), <anonymous>:42:12)
    at eval (eval at run_py (http://localhost:63342/Github/cfn-lint/brython.js:8807:14), <anonymous>:6471:2)
    at run_py (http://localhost:63342/Github/cfn-lint/brython.js:8807:41)
    at exec_module (http://localhost:63342/Github/cfn-lint/brython.js:8919:41)
    at f (http://localhost:63342/Github/cfn-lint/brython.js:6267:30)
    at method (http://localhost:63342/Github/cfn-lint/brython.js:6271:58)
brython.js:8812 args ["No module named six", __brython__: true, __class__: {…}]
brython.js:8812 __class__ {__class__: {…}, __mro__: Array(3), $is_class: true, $infos: {…}, $factory: ƒ, …}
brython.js:8812 $py_error true
brython.js:8812 $stack (2) [Array(4), Array(4)]
brython.js:8812 $line_info undefined
brython.js:8812 __cause__ {__class__: {…}}
brython.js:8812 __context__ {__class__: {…}}
brython.js:8812 __suppress_context__ false
brython.js:8812 name six
brython.js:8813 Traceback (most recent call last):
brython.js:8814 message: undefined
brython.js:8815 filename: undefined
brython.js:8816 linenum: undefined
brython.js:5979 Traceback (most recent call last):
ImportError: No module named six
```

The only top-level dependency I am trying to import is [cfnlint](https://github.com/aws-cloudformation/cfn-python-lint)

I initially tried:
`python3 -m brython --add_package cfnlint`
but also tried this after seeing this error as well:
`python3 -m brython --add_package six`

It seems like it might be located here after explicitly adding the package `six`:
`Lib/site-packages/six/six.py`

**Comment by PierreQuentel** [comment]
Thanks for the report. I don't know this cfnlint package, but I see that it requires packages like requests and urllib3, so it's very likely that it can't run in the browser (cf [this thread](https://groups.google.com/forum/?fromgroups=#!topic/brython/vZ7lNMp5Qlk) in the Google Group).

Anyway, there were issues in the brython CPython package, that are adressed (but not completely fixed) by commits 0c63e4bfc38eee998164cd03621afce0bc9f28d7 and 7f5992d348a436563bb121a439c1af3aca05b73a.

**Comment by PatMyron** [comment]
Thanks for your quick response and all of your work on Brython!

`requests` was actually removed in the most recent release, but it does still use `urllib`:
https://github.com/aws-cloudformation/cfn-python-lint/commit/96150780ce3d8cb543baeef3360cba735cef4e4a
https://github.com/aws-cloudformation/cfn-python-lint/releases/tag/v0.23.3

While they are in the package, I do not need their functionality and am not planning on calling the codepath that would use either. Is that sufficient, or will the whole package fail to import regardless?

This stacktrace seems to be failing to import `six`, which seems like it was supported in some capacity: https://github.com/brython-dev/brython/issues/777
Maybe @aparamon can help with an example of how to import `six`

Is there a way to check which dependencies from pip are compatible with Brython?
